### PR TITLE
wip: run hive on gnosis

### PR DESCRIPTION
**Pull request by revitteth**
- [ ] allow debug of simulator
- [ ] provide engine genesis for erigon/nethermind (mapper alterations also required)
- [ ] add hooks for withdrawals suite to alter testnet config for gno

### CODE REVIEW

File: clients/erigon/genesis.json [hunk 1]
```diff
@@ -7,9 +7,177 @@
     "nonce"      : "0x78cc16f7b4f65485",
     "parentHash" : "0x0000000000000000000000000000000000000000000000000000000000000000",
     "timestamp"  : "0x54c98c81",
+    "config": {
+        "aura": {
```
Comment (revitteth) [inline clients/erigon/genesis.json:11]: This config isn't supported by the go-ethereum library being imported.

**Comment by 4rgon4ut** [comment]
@revitteth Thanks for have your hands on this! Im sorry this PR are currently stale, can we close it?
/cc @stdevMac

**Comment by stdevMac** [comment]
Thanks so much for your help @revitteth 🙏

**Comment by revitteth** [comment]
> Thanks so much for your help @revitteth 🙏 

Anytime!

**Pull request closed without merging**